### PR TITLE
Allow the modernizr-bundle to not be a chunk

### PR DESCRIPTION
**Pull request by SimenB**
Last one :laughing: 

If `asEntry` (I'm awesome at names, I know) is set to false, the file is not added to `chunks` (and not iterated over in the default templates), but as it's own entry. Then it can be put wherever the user wants, e.g. in `<head>`.

Handlebars-sample:
```hbs
<script src="{{htmlWebpackPlugin.files.modernizr-bundle}}"></script>
```

Again, unsure if this is the correct implementation, but it works for our project.

If you want it, I'll document it as well :smile:. And write a test if applicable

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -25,23 +25,28 @@ function ModernizrPlugin(options) {
...
-    };
-    // get html-webpack-plugin to output modernizr chunk first
-    result.chunks = assign({}, chunk, result.chunks);
+        css: [],
+        size: filesize || 0
+      };
+      // get html-webpack-plugin to output modernizr chunk first
+      result.chunks = assign({}, chunk, result.chunks);
+    } else {
+      result['modernizr-bundle'] = oFilename;
```
Comment (SimenB) [inline index.js]: should probably use `filename`

**Comment by alexpalombaro** [comment]
@SimenB I don't fully understand this one. 

I thought if you want to have your scripts appear in the head you use the [html-webpack-plugin](https://www.npmjs.com/package/html-webpack-plugin) option `{inject:'head'}` or if you wanted to specify the exact position you would use the template options then include it with:

`<script src="{%=o.htmlWebpackPlugin.files.chunks.modernizr-bundle.entry %}"></script>`

This is how I read [documentation](https://www.npmjs.com/package/html-webpack-plugin)... is there another templating feature I'm missing?

Anyway in my mind the more options the merrier. If you can update the doco with a sample maybe it will help me understand this cool new feature :smiley:

**Comment by SimenB** [comment]
@alexpalombaro I'm using my own template, not the default one. And I only want the modernizr bundle in `head`, my app code I want in body.

For my own template, I loop over all chunks and add the entry points. I don't know the name of it, as it's not static (I want to add the hash of the file to enable long term caching).

**Comment by alexpalombaro** [comment]
@SimenB I think I've got where you are going with this feature and have tried to implement it and document in the README.md (changed from asEntry to noChunk because to me its a change from default behaviour and passing a negative to turn *on* a feature makes me :dizzy: )

If I've totally missed the point please feel free to educate me further :smiley:

**Pull request merged**